README.md

```diff
@@ -1,10 +1,10 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task-move-spider-DOM/)
+    - [DEMO LINK](https://hetskoweb.github.io/js_task-move-spider-DOM/)
```

Comment:
Add this demo link to the PR description 
<img width="941" alt="image" src="https://github.com/user-attachments/assets/45ed6cda-2c57-4f71-84eb-439181c32e8a" />